### PR TITLE
Update Blazor Project to use MudBlazor

### PR DESCRIPTION
**Pull request by cptcrunchy**
Imported the Mudblazor Nuget Package.
Removed bootstrap css and classes
Restructured layout to use MudBlazor components.

### CODE REVIEW

File: src/Congo.Blazor/Shared/NavMenu.razor [hunk 1]
```diff
@@ -1,29 +1,16 @@
...
-            </NavLink>
-        </li>
-    </ul>
-</div>
+﻿<MudNavMenu>
+	<MudText Typo="Typo.h6" Class="px-4">My Application</MudText>
+	<MudText Typo="Typo.body2" Class="px-4 mud-text-secondary">Secondary Text</MudText>
+	<MudDivider Class="my-2" />
+	<MudNavLink Href="/dashboard">Dashboard</MudNavLink>
+	<MudNavLink Href="/servers">Servers</MudNavLink>
```
Comment (pobiega) [inline src/Congo.Blazor/Shared/NavMenu.razor]: Servers? Billing? Users? Security?

We don't have any of these either as planned features or currently requested features. We do however have the whole products subtree.

File: Congo.sln [hunk 2]
```diff
@@ -5,11 +5,11 @@ VisualStudioVersion = 16.0.31205.134
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{4AE32A8E-6CBE-4ED6-AAC2-8B15CC50EC03}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Congo.WebApi", "src\Congo.WebApi\Congo.WebApi.csproj", "{B99C12F3-BE32-458E-81F7-040DC27171EB}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Congo.WebApi", "src\Congo.WebApi\Congo.WebApi.csproj", "{B99C12F3-BE32-458E-81F7-040DC27171EB}"
```
Comment (pobiega) [inline Congo.sln]: Classic VS problem where it randomly decides to generate a new GUID for these. If you can discard this files changes and it still works, I would prefer to keep the diffs clean and avoid modifying files that don't need to be, but I know VS tends to repeat the same changes forever until it gets its way.

Give it a try at least.

**Review by pobiega — CHANGES_REQUESTED** [review]

File: src/Congo.Blazor/Shared/NavMenu.razor (hunk 1 above)
Comment (cptcrunchy) [reply src/Congo.Blazor/Shared/NavMenu.razor]: These were from the example code.

File: Congo.sln (hunk 2 above)
Comment (cptcrunchy) [reply Congo.sln]: I removed this from the commit. Should still work with the original GUID. VS can be weird sometimes.

**Review by pobiega — APPROVED** [review]

**Pull request merged**